src/app/component/user/modal-auth/windowsignin/windowsignin.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="form-wrapper">
+    <a (click)="close()" class="close-icon">&#10005;</a>
+    <div class="form-pic "><img [src]="formPic"></div>
+    <div class="sign-in">
+        <form #registerForm="ngForm" (ngSubmit)="signIn(userOwnSignIn)">
+            <h1>{{'user.auth.sign-in.title' | translate}}</h1>
+            <h2>{{'user.auth.sign-in.fill-form' | translate}}</h2>
+            <label for="email">{{'user.auth.sign-in.email' | translate}}</label>
+            <input id="email" #emailControl="ngModel" type="email" class="sign-in-input width-inp" [ngClass]="emailControl.errors&&emailControl.touched? 'red':'grey'" name="email" [(ngModel)]="userOwnSignIn.email" required minlength="1" email="true" appInputcolor [typeColor]="'#13AA57'" [defaultColor]="'#000'">
+            <div class="validation-error" *ngIf="emailControl.invalid && (emailControl.dirty||emailControl.touched)">
+                <div *ngIf="emailControl.errors.required">
+                    {{'user.auth.sign-in.email-is-required' | translate}}
+                </div>
+                <div *ngIf="emailControl.errors.minlength">
+                    {{'user.auth.sign-in.email-must-be-at-least-1-characters-long' | translate}}
+                </div>
+                <div *ngIf="emailControl.errors.email">
+                    {{'user.auth.sign-in.this-is-not-email' | translate}}
+                </div>
+            </div>
+            <div class="validation-error" *ngIf="emailErrorMessageBackEnd!=null">
+                {{emailErrorMessageBackEnd}}
+            </div>
+            <label for="password">{{'user.auth.sign-in.password' | translate}}</label>
+            <div class="password-wrapper">
+                <input id="password" #passwordControl="ngModel" type="password" class="sign-in-input" name="form-control password-signin" [(ngModel)]="userOwnSignIn.password" [ngClass]="passwordControl.errors&&passwordControl.touched? 'red':'grey'" required minlength="8" appInputcolor [typeColor]="'#13AA57'" [defaultColor]="'#000'">
```

Comment:
Bad readability: if you have more than 2 attributes in tag name they should be located in one column

Reply:
looks like this ?  http://i.piccy.info/i9/274b10e603f54625f3f1ce68f1c9ae80/1585128600/98589/1368079/column.jpg

Reply:
Yes, it should be in one column if there're more than 2 attributes

Reply:
ready